HCP TF SSO account linking (#717)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/users.mdx

<!-- BEGIN: TFC:only -->

## Log in with a HashiCorp Cloud Platform account

We recommend using a [HashiCorp Cloud Platform (HCP)](https://portal.cloud.hashicorp.com/sign-up) account to log in to HCP Terraform. Your HCP Account grants access to every HashiCorp product and the Terraform Registry. If you use an HCP Account, you manage account settings like multi-factor authentication and password resets from within HCP instead of the HCP Terraform UI.

To log in with your HCP account, go to the [Sign In to HCP Terraform](https://app.terraform.io/) page and click **Continue with HCP account**. HCP Terraform may ask if you want to link your account.

### Linking HCP and HCP Terraform accounts

The first time you log in with your HCP credentials, HCP Terraform searches for an existing HCP Terraform account with the same email address. If you have an unlinked account, HCP Terraform asks if you want to link it to your HCP account. Otherwise, if no account matches your HCP account's email address, HCP Terraform creates and automatically links a new HCP Terraform account to your HCP account.

> !> **Compatibility warning**: After linking your HCP and HCP Terraform accounts, you can only log in with your HCP credentials. We do not recommend linking your account if you use an SSO provider to log in to HCP Terraform.

Author: ```suggestion
> **Note**: After linking your HCP and HCP Terraform accounts, you can only log in with your HCP credentials. We do not recommend linking your account if you use an SSO provider to log in to HCP Terraform because linking your account may conflict with your existing SSO configuration. 
```

> The only way to log in with your old HCP Terraform credentials is to unlink your HCP Terraform and HCP accounts. For more details, refer to [HCP Account Linking](#hcp-account-linking).

Author: ```suggestion
The only way to log in with your old HCP Terraform credentials is to unlink your HCP Terraform and HCP accounts. For more details, refer to [HCP Account Linking](#hcp-account-linking). If HCP Terraform generated an account for you, you cannot unlink that account from your HCP account. You can unlink a pre-existing HCP Terraform account on the [HCP Account Linking page](#hcp-account-linking) in your **Account settings**.
```

> 

Author: ```suggestion
```

> If HCP Terraform generated an account for you, you cannot unlink that account from your HCP account. You can unlink a pre-existing HCP Terraform account on the [HCP Account Linking page](#hcp-account-linking) in your **Account settings**.

Author: ```suggestion
```
